src/models/order.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model, SchemaTypes } = require("mongoose");
+
+const schema = new Schema({
+    productionDay: {
+        type: Date,
+        required: true,
+        unique: true,
+        length: 7
```

Comment:
Only use unique when necessary. I can already tell you that given how Date type is stringified, you will enver satisfy this condition.